testing/tests/install-invalid

```diff
@@ -0,0 +1,12 @@
+# Test invalid package names
+# @TEST-EXEC: ! bash %INPUT 2>output
```

Comment:
```suggestion
# @TEST-EXEC-FAIL: bash %INPUT 2>output
```

Think that's a bit clearer and just going to change in the merge, but thought to mention in case you didn't know about it (or just let me know if you had other reason to prefer the original way).

Reply:
Perfect, wasn't aware this existed - thanks for letting me know!